Slightly changed the note on protocols (#849)
SoftwareAG/c8y-docs: content/users-guide/device-management-bundle/connecting-devices.md

To register devices, you can select one of the following options:

> * **[Single device registration](#device-registration-manually)** - to manually connect devices one by one.

Reviewer: "one by one" to me reads like there must be more than one, where the line before the change allowed "just one" as well :thinking:

Reviewer: i guess that would work :)

> * **[Single device registration](#device-registration-manually)** - to manually connect devices one by one.

Author: ```suggestion
* **[Single device registration](#device-registration-manually)** - to manually connect one device or several devices one by one.
```

* **[Bulk device registration](#bulk-registration)** - to register larger amounts of devices in one step.

Microservice developers can also use the [Extensible device registration](/concepts/applications/#extensible-device-registration) and implement a custom registration form that blends seamlessly into the UI.

{{< c8y-admon-info >}}

> The following descriptions apply to the general device registration processes. If you subscribe to specific protocol integrations, you will see additional protocol-specific options (for example, for LWM2M or OPC UA). A full list of supported protocols can be found in the [Protocol integration guide](/protocol-integration/overview/). There you will also find descriptions for the protocol specific registration processes.

Reviewer: ```suggestion
The following descriptions apply to the general device registration processes. If you subscribe to specific protocol integrations, you will see additional protocol-specific options (for example, for LWM2M or OPC UA). A full list of supported protocols can be found in the [Protocol integration guide](/protocol-integration/overview/). It also contains descriptions for the protocol specific registration processes.
```

to avoid "will"
